resources/[managers]/spawnmanager/spawnmanager.lua

```diff
@@ -248,24 +248,25 @@ function spawnPlayer(spawnIdx, cb)
 
         -- if the spawn has a model set
         if spawn.model then
-            RequestModel(spawn.model)
-
-            -- load the model for this spawn
-            while not HasModelLoaded(spawn.model) do
+            if IsModelValid(spawn.model) then
```

Comment:
Should be add a warning or something when the model isn't valid ?

Reply:
That's what it does in the else clause for that if.